Upstream: yes
Reason: The test is broken because the content includes changing data
Url: https://github.com/betamaxpy/betamax/issues/184
diff --git a/tests/integration/test_record_modes.py b/tests/integration/test_record_modes.py
index 58c8846..7f206cd 100644
--- a/tests/integration/test_record_modes.py
+++ b/tests/integration/test_record_modes.py
@@ -40,7 +40,8 @@ class TestRecordOnce(IntegrationHelper):
             # NOTE(hroncok): httpbin.org added X-Processed-Time header that
             # can possibly differ (and often does)
             assert r0_headers == r1_headers
-            assert r0.content == r1.content
+            # Fails due to X-Amzn-Trace-Id being different on every request
+            # assert r0.content == r1.content
 
 
 class TestRecordNone(IntegrationHelper):
